### ISSUE TITLE
Reproduce of mIOU on ADE20K

### ISSUE THREAD

**Issue by WeilunWang** [boschresearch/OASIS#17]
Hello!
Thank you for the avaliable code!
I'm trying to reproduce the results for ADE20K. For FID, I use the pretrained model and get a similar results in the paper.
However, for mIOU, I used upernet101 on the previously generated data and get 42.76 mIOU, which has a little gap with that reported the paper. I think this is caused by the different testing configuration. I want to make sure wheter you use the following settings when testing. 
`DATASET:
  root_dataset: "./results/oasis_ade20k_pretrained/best/"
  list_train: "./data/training.odgt"
  list_val: "./data/validation_oasis.odgt"
  num_class: 150
  imgSizes: (256)
  imgMaxSize: 1000
  padding_constant: 32
  segm_downsampling_rate: 4
  random_flip: True

MODEL:
  arch_encoder: "resnet101"
  arch_decoder: "upernet"
  fc_dim: 2048

TRAIN:
  batch_size_per_gpu: 2
  num_epoch: 40
  start_epoch: 0
  epoch_iters: 5000
  optim: "SGD"
  lr_encoder: 0.02
  lr_decoder: 0.02
  lr_pow: 0.9
  beta1: 0.9
  weight_decay: 1e-4
  deep_sup_scale: 0.4
  fix_bn: False
  workers: 16
  disp_iter: 20
  seed: 304

VAL:
  visualize: False
  checkpoint: "epoch_50.pth"

TEST:
  checkpoint: "epoch_50.pth"
  result: "./"

DIR: "ckpt/ade20k-resnet101-upernet"
`
The content of the file oasis.odgt is as follow:
{'fpath_img': 'image/ADE_val_00000502.png', 'fpath_segm': 'label/ADE_val_00000502.png', 'width': 256, 'height': 256}

Thank you!

**Comment by SushkoVadim** [comment]
Hi,

I hope the problem is solved by now. Here is the config file to compare to:
```
DATASET:
  root_dataset: "./results/oasis_ade20k_pretrained/best/"
  list_train: "./data/training.odgt"
  list_val: "./data/validation.odgt"
  num_class: 150
  imgSizes: (300, 375, 450, 525, 600)
  imgMaxSize: 1000
  padding_constant: 32
  segm_downsampling_rate: 4
  random_flip: True

MODEL:
  arch_encoder: "resnet101"
  arch_decoder: "upernet"
  fc_dim: 2048

TRAIN:
  batch_size_per_gpu: 2
  num_epoch: 40
  start_epoch: 0
  epoch_iters: 5000
  optim: "SGD"
  lr_encoder: 0.02
  lr_decoder: 0.02
  lr_pow: 0.9
  beta1: 0.9
  weight_decay: 1e-4
  deep_sup_scale: 0.4
  fix_bn: False
  workers: 16
  disp_iter: 20
  seed: 304

VAL:
  visualize: False
  checkpoint: "epoch_50.pth"

TEST:
  checkpoint: "epoch_50.pth"
  result: "./"

DIR: "ckpt/ade20k-resnet101-upernet"
```

The only difference here seems to be the multi-scale regime with multiple resolutions (`imgSizes`) (I think this was the default setting)
